.github/workflows/ci.yml

```diff
@@ -33,9 +36,20 @@ jobs:
         run: |
           crystal spec
           crystal spec --release --no-debug
+        continue-on-error: ${{ matrix.allowed-failure }}
 
       - name: Check formatting
         run: crystal tool format --check
```

Comment:
This fails on Windows, not sure why. It works fine locally.
I would suggest to disable this spec on windows. Actually, it only needs to execute once - the expected result is the same on any runner (`if: matrix.os == 'ubuntu-latest' && matrix.crystal == 'nightly'`)